### PR TITLE
Unstructured parser: Support txt

### PR DESCRIPTION
**Pull request by flash1293**
Built on https://github.com/airbytehq/airbyte/pull/32700

This PR adds support for the TXT file type by simply passing files through as a string. Files with a mime type of `text/plain` are also recognized as TXT, so this allows to extract "regular" text fields as strings.

### CODE REVIEW

**Review by aaronsteers — COMMENTED** [review]
I'm not quite sure yet about json - whether we should pass as raw json or "markdownify/yaml-fy" it.

No questions on txt though - that's an easy inclusion.

File: airbyte-cdk/python/unit_tests/sources/file_based/scenarios/unstructured_scenarios.py [hunk 1]
```diff
@@ -131,7 +131,7 @@
         FileBasedSourceBuilder()
         .set_files(
             {
-                "a.txt": {
```
Comment (clnoll) [inline airbyte-cdk/python/unit_tests/sources/file_based/scenarios/unstructured_scenarios.py:134]: Mind adding two basic test scenarios - one with a txt file and one with json, showing that the unstructured parser handles them and gives the expected output?

**Comment by flash1293** [comment]
@aaronsteers @clnoll AJ and me discussed this offline and decided to remove JSON for now, does this get approval from your side based on that?

**Comment by clnoll** [comment]
@flash1293 ideally I'd like a scenario-based test that includes a txt file.

**Comment by flash1293** [comment]
@clnoll Of course, good call! Added that

**Review by clnoll — APPROVED** [review]
Thanks @flash1293!

**Pull request merged**